components/plan-build/plan.sh

```diff
@@ -18,6 +18,7 @@ pkg_deps=(
   core/grep
   core/gzip
   core/hab
+  core/rq
```

Comment:
Could you add a check for this command in `hab-plan-build.sh`'s [`_find_system_commands()`](https://github.com/habitat-sh/habitat/blob/2aae1efc68da24389c1accb096b5f5df8395074d/components/plan-build/bin/hab-plan-build.sh#L495-L513) function? This function becomes important on net-new target systems in the future as they are special programs that won't necessarily appear on a vanilla system and are needed for the build program to be well behaved before a Studio environment is possible.